website/docs/addon_resolve_artist.md

```diff
@@ -114,7 +114,9 @@ After all clips which are intended to be converted to publishable instances are
 
 After the menu widget is opened (it can take while so be patient please :).
 
-Hit `Create ...` and then set **Use selection** to active and select the family to **Create Publishable Clips**.
+Hit `Create ...` and then select the family to **Create Publishable Clip**.
+
+You can leave `Use only clips with Chocolate clip colors` to publish only the selection.
```

Comment:
Did you mean to mention to leave it enabled?
```suggestion
You can leave `Use only clips with Chocolate clip colors` enabled to publish only the selection.
```

Reply:
Does the word `family` in this context is related to resolve or AYON?
should it be ? 
```
Hit `Create ...` and then select the publish type **Create Publishable Clip**.
```

Reply:
It should be `product type` actually you are correct.